dev/MenuFlyout/MenuFlyout_themeresources.xaml

```diff
@@ -941,9 +914,6 @@
                                         <Setter Target="CheckGlyph.Foreground" Value="{ThemeResource ToggleMenuFlyoutItemCheckGlyphForegroundPressed}" />
                                         <contract6Present:Setter Target="KeyboardAcceleratorTextBlock.Foreground" Value="{ThemeResource ToggleMenuFlyoutItemKeyboardAcceleratorTextForegroundPressed}" />
                                     </VisualState.Setters>
-                                    <Storyboard>
-                                        <PointerDownThemeAnimation Storyboard.TargetName="AnimationRoot" />
```

Comment:
is the AnimationRoot grid even needed anymore?

Reply:
I think so, it has two columns for Icon and Text

Reply:
right, which can be moved to the parent grid. the other visual color states are on the parent grid

Reply:
@beervoley will be following up with a separate PR to refactor that.